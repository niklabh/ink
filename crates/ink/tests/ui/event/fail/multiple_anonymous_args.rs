#[ink::event]
#[ink(anonymous)]
#[ink(anonymous)]
pub struct Event {
    #[ink(topic)]
    pub topic: [u8; 32],
}

fn main() {}