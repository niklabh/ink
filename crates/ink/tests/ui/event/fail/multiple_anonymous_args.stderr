error: Only a single `#[ink(anonymous)]` attribute allowed.
 --> tests/ui/event/fail/multiple_anonymous_args.rs:3:1
  |
3 | #[ink(anonymous)]
  | ^

error[E0277]: the trait bound `Event: ink::ink_env::Event` is not satisfied
 --> tests/ui/event/fail/multiple_anonymous_args.rs:2:1
  |
2 | #[ink(anonymous)]
  | ^ the trait `ink::ink_env::Event` is not implemented for `Event`
